rules/windows/registry_event/sysmon_wab_dllpath_reg_change.yml

```diff
@@ -0,0 +1,24 @@
+title: Execution DLL of Choice Using WAB.EXE
+id: fc014922-5def-4da9-a0fc-28c973f41bfb
+description: This rule detects that the path to the DLL written in the registry is different from the default one. Launched WAB.exe tries to load the DLL from Registry.
+status: experimental
+references:
+    - https://github.com/LOLBAS-Project/LOLBAS/blob/master/yml/OSBinaries/Wab.yml
+    - https://twitter.com/Hexacorn/status/991447379864932352
+    - http://www.hexacorn.com/blog/2018/05/01/wab-exe-as-a-lolbin/
+tags:
+    - attack.defense_evasion
+    - attack.t1218
+date: 2020/10/13
+author: oscd.community, Natalia Shornikova
+logsource:
+    category: registry_event
+    product: windows
+detection:
+    selection:
+        TargetObject|endswith: '\Software\Microsoft\WAB\DLLPath'
+    filter:
+        Details: '%CommonProgramFiles%\System\wab32.dll'
```

Comment:
Hello @nsaddler! I think that this construction is not gonna work. Have you tried to convert your rule into a query?

Reply:
Hello, sorry for the delay!
Why not gonna work?
ElasticSearch query:
(winlog.event_data.TargetObject.keyword:*\\Software\\Microsoft\\WAB\\DLLPath AND (NOT (winlog.event_data.Details:"%CommonProgramFiles%\\System\\wab32.dll")))

Reply:
Hello @nsaddler!

No problems (: 

>>>     Details: '%CommonProgramFiles%\System\wab32.dll'
>>      I think that this construction is not gonna work.
> Why not gonna work?

I was thinking that you are using the Sigma [placeholders](https://github.com/Neo23x0/sigma/wiki/Specification#placeholders) feature here.
But now I see that it is an original string from the [hexacorn's blogpost](http://www.hexacorn.com/blog/2018/05/01/wab-exe-as-a-lolbin/) and related to the registry events.

Sorry about that. Now I get it (: